src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Resources/views/translation.html.php

```diff
@@ -46,4 +46,4 @@
 
 <?php echo $view['translator']->trans('typecast', ['a' => (int) '123'], 'not_messages'); ?>
 <?php echo $view['translator']->transChoice('msg1', 10 + 1, [], 'not_messages'); ?>
-<?php echo $view['translator']->transChoice('msg2', intval(4.5), [], 'not_messages'); ?>
+<?php echo $view['translator']->transChoice('msg2', (int)4.5, [], 'not_messages'); ?>
```

Comment:
Hm, I wondering: what is the intention of using the function here, would you share the  idea with me please?